AntiCovid/app/src/main/java/com/swtug/anticovid/login/LoginFragment.kt

```diff
@@ -75,6 +76,12 @@ class LoginFragment : BaseFragment() {
 
             }
         })
+        val callback: OnBackPressedCallback = object : OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
+                // User is not allowed to go back
+            }
+        }
+        requireActivity().onBackPressedDispatcher.addCallback(requireActivity(), callback);
```

Comment:
Wouldnt the app just close if a user back presses on Login Fragment? I think its a question of how we want it to be or were ther other issues?

Reply:
Nothing will happen when the back button is pressed. This was the only solution that we found which is working.

Reply:
For me there are two cases what could happen if i press the back button in the login screen
1. Go to Terms of Use Screen
2. Close App

But i dont think that nothing should happen.

Furthermore this should be handled in the MainActivity